baselines/commoncontrols.md

```diff
@@ -1118,7 +1118,7 @@ To configure Supplemental Data Storage per the policy:
 
 ## 16. Additional Google Services
 
-This section covers the Google services that do not have an individual control and whether these services are on or off.
+This section covers the Google services that do not have an individual control and whether these services are on or off. This section also covers whether users have access to early access apps.
```

Comment:
Might I suggest the following? Not all readers will be familiar with what "additional Google services" means. The intro of the group is an ideal place to provide some context for the reader.
```suggestion
Google Workspace considers some of its services "core services," including Gmail, Calendar, and Drive. Services outside of this core offering are controlled within the "Additional Google services" portion of the admin console. This section outlines requirements relating to those services.
```